google/cloud/storage/_signing.py

```diff
@@ -20,7 +20,8 @@
 import hashlib
 import json
 
-import six
+import http
```

Comment:
Prefer to import the explicit submodule:
```suggestion
import http.client
```

Also, sort w/ stdlib imports.